datasets/poleval2019_mt/poleval2019_mt.py

```diff
@@ -0,0 +1,206 @@
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = ""
+
+
+_DESCRIPTION = """\
+PolEval is a SemEval-inspired evaluation campaign for natural language processing tools for Polish.\
+Submitted solutions compete against one another within certain tasks selected by organizers, using available data and are evaluated according to\
+pre-established procedures. One of the tasks in PolEval-2019 was Machine Translation (Task-4).\
+
+The task is to train as good as possible machine translation system, using any technology,with limited textual resources.\
+The competition will be done for 2 language pairs, more popular English-Polish (into Polish direction) and pair that can be called low resourced\
+Russian-Polish (in both directions).
+
+Here, Polish-English is also made available to allow for training in both directions. However, the test data is ONLY available for English-Polish.
+"""
+
+# Official homepage for the dataset
+_HOMEPAGE = "http://2019.poleval.pl/"
+
+# Licence
+_LICENSE = ""
+
+# All the tsv files are present in the below link.
+_TEST_URL = "http://2019.poleval.pl/task4/task4_test.zip"
+
+# These are the supported languages in the parallel corpora in the PolEval-2019 MT task
+_SUPPORTED_LANGUAGES = {
+    "ru": "Russian",
+    "en": "English",
+}
+
+
+class PolevalMTConfig(datasets.BuilderConfig):
+    """BuilderConfig for PolEval-2019 MT corpus."""
+
+    def __init__(self, language_pair=(None, None), **kwargs):
+        """BuilderConfig for PolEval-2019.
+        Args:
+            for the `datasets.features.text.TextEncoder` used for the features feature.
+            language_pair: pair of languages that will be used for translation. Should
+            contain 2-letter coded strings. First will be used at source and second
+            as target in supervised mode. For example: ("pl", "en").
+          **kwargs: keyword arguments forwarded to super.
+        """
+        # Validate language pair.
+        name = "%s-%s" % (language_pair[0], language_pair[1])
+        assert "pl" in language_pair, ("Config language pair must contain `pl` (Polish), got: %s", language_pair)
+        source, target = language_pair
+        non_pl = source if target == "pl" else target
+        assert non_pl in _SUPPORTED_LANGUAGES.keys(), ("Invalid non-polish language in pair: %s", non_pl)
+
+        description = ("Translation dataset between Polish and %s") % (_SUPPORTED_LANGUAGES[non_pl])
+        super(PolevalMTConfig, self).__init__(
+            name=name,
+            description=description,
+            version=datasets.Version("1.0.0", ""),
+            **kwargs,
+        )
+
+        self.language_pair = language_pair
+
+
+class Poleval2019Mt(datasets.GeneratorBasedBuilder):
+    """Polish Translation Dataset"""
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+    \n    You need to go to https://drive.google.com/drive/folders/1nl98B8I_xlKa_t4RFOLmIkCVuF-WEe4B,
```

Comment:
Did you try using the download manager using those urls ?
- dev.en: https://drive.google.com/u/0/uc?id=1L6qQiO6kPLFj8BUK9XFNUH7bNyJVA7FC&export=download
- dev.pl: https://drive.google.com/u/0/uc?id=1CP3oHL04qE1nfu3h_zmaxz5fmEtlwzLs&export=download
- train.en: https://drive.google.com/u/0/uc?id=1NAeuWLgYBzLwU5jCdkrtj4_PRUocuvlb&export=download
- train.pl: https://drive.google.com/u/0/uc?id=13ZyFc2qepAYSg9WIFaeJ9y402gblsl2e&export=download

It would be convenient to not have to download them by hand

Reply:
I had tried putting everything for auto download but train.en was repeatedly giving connection error when I tried with download manager. I already have the code ready, shall I push that for verification from your side?

Reply:
PyTests and CircleCI is failing. I have just changed download from manual to automatic. Please review as to what can be done.